tests/30rooms/06invite.pl

```diff
@@ -87,6 +87,9 @@ sub inviteonly_room_fixture
          $content->{membership} eq "invite" or
             die "Expected membership to be 'invite'";
 
+         require_json_keys( $event, qw( invite_room_state ));
```

Comment:
invite_room_state is optional; we accordingly shouldn't require this.

Opened https://github.com/matrix-org/matrix-doc/pull/160 to clarify this.